ang/civicase/activity/feed/directives/activity-feed.directive.js

```diff
@@ -414,7 +414,7 @@
       } else if (!$scope.displayOptions.include_case) {
         params.case_id = { 'IS NULL': 1 };
       } else {
-        returnParams.return = returnParams.return.concat(['case_id.case_type_id', 'case_id.status_id', 'case_id.contacts']);
+        returnParams.return = returnParams.return.concat(['case_id.case_type_id', 'case_id.status_id']);
```

Comment:
Why did we change this logic? I have found a bug which might be related to this. Create a draft email activity. And try to access the same from the Dashboard Activity tab, now click on the activity to open the activity panel. there is an error saying "Email address is not configured". But this is working fine with master branch.

Reply:
Also could you verify if you can access Activity Details panel in the Contact Activity Tab for normal activities and also Draft email activities(with `Show case activities` option enabled/disabled).

Reply:
Checking